loadtest/control/actions.go

```diff
@@ -729,3 +729,33 @@ func Reload(u user.User) UserActionResponse {
 
 	return UserActionResponse{Info: "page reloaded"}
 }
+
+// MessageExport simulates the given user performing
+// a compliance message export
+func MessageExport(u user.User) UserActionResponse {
+	isAdmin, err := u.IsSysAdmin()
+	if err != nil {
+		return UserActionResponse{Err: NewUserError(err)}
+	}
+
+	if !isAdmin {
+		return UserActionResponse{Info: "user is not a sysadmin and cannot perform a message export"}
+	}
+
+	if err := u.GetConfig(); err != nil {
+		return UserActionResponse{Err: NewUserError(err)}
+	}
+
+	cfg := u.Store().Config()
+
+	if &cfg != nil && !*cfg.MessageExportSettings.EnableExport {
```

Comment:
I actually meant `cfg.MessageExportSettings.EnableExport != nil`. We use pointers for the config values (not ideal but that comes from the server).
`cfg` itself is safe since we return a value.